src/main/java/io/fabric8/jenkins/openshiftsync/PodTemplateUtils.java

```diff
@@ -258,6 +268,26 @@ protected static boolean isReservedPodTemplateName(String name) {
       return results;
   }
 
+  protected static PodTemplate updatePodTemplateEnvVars(HashMap<String,String> map, PodTemplate podTemplate){
+      List<PodEnvVar> envVars = podTemplate.getEnvVars();
+
```

Comment:
It would be safe to add a nullity check for envVars List.

Reply:
done